server/app.ts

```diff
@@ -30,6 +31,9 @@ export default function createApp(controllers: Controllers, services: Services):
   app.set('trust proxy', true)
   app.set('port', process.env.PORT || 3000)
 
+  // Add method-override to allow us to use PUT and DELETE methods
+  app.use(methodOverride('_method'))
```

Comment:
just for my understanding... why do we need to override PUT and DELETE?

Reply:
Again, this is all due to me being a REST obsessive. We can't use PUT or DELETE in browsers, so this lets us simulate those requests. This comes for free in Rails, so this is just to keep consistent with that way of doing things

Reply:
Good to know, thanks!